script/Arcueid/Day 3/03_00_ARC03_6.txt

```diff
@@ -147,22 +148,23 @@ Hisui slightly raises her eyebrows.
 [sha:5845398749ee9544235a829278ab135645cb9e66]{
 -- Page 281, Offset 4771.
 -- 「はい。志貴さまはベンチで休まれており、その後、ご自分の足で屋敷までお帰りになられました」
-"Yes, you were resting on a bench. After my sister found you, you walked back to the residence on your own two feet."
+"Yes. You had taken a moment to rest on the bench, and afterward returned to the mansion on your own two feet."
```

Comment:
despite the redundancy of "you" "you" "your", i think it reads cleaner to me as "afterward you returned"